Src/WitsmlExplorer.Frontend/contexts/navigationActions.tsx

```diff
@@ -1,7 +1,7 @@
 import BhaRun from "../models/bhaRun";
 import LogObject from "../models/logObject";
 import MessageObject from "../models/messageObject";
-import MudLogObject from "../models/mudLog";
+import { default as MudLog, default as mudLog } from "../models/mudLog";
```

Comment:
🤔

Reply:
I think prettier did this. (auto imports). Idk why

Reply:
Fixed